main.go

```diff
@@ -133,8 +129,9 @@ func main() {
 			},
 			determineTokenAcquirer,
 			webhookClient.NewBasicRegisterer,
-			func(l fx.Lifecycle, r *webhookClient.BasicRegisterer, c WebhookConfig, logger log.Logger) (*webhookClient.PeriodicRegisterer, error) {
-				return webhookClient.NewPeriodicRegisterer(r, c.RegistrationInterval, logger, provider.NewDiscardProvider())
+			webhookClient.NewProvideMeasures,
+			func(l fx.Lifecycle, r *webhookClient.BasicRegisterer, c WebhookConfig, m *webhookClient.Measures, logger log.Logger) (*webhookClient.PeriodicRegisterer, error) {
+				return webhookClient.NewPeriodicRegisterer(r, c.RegistrationInterval, logger, m)
```

Comment:
This might make sense as a `Provide()` function in wrp-listener? What do you think?

Reply:
yeah that makes sense! I'll add it to wrp-listener.